### ISSUE TITLE
Add overall N to count_layer?

### ISSUE THREAD

**Issue by Robert-Krajcik** [atorus-research/Tplyr#11]
I have a table I am trying to produce with Tplyr
This happens to be a Phase-1 study, so N=8 in each column

![image](https://user-images.githubusercontent.com/75215958/120808061-aac63180-c516-11eb-9493-58bdcb7a4d91.png)

Is there, or can there be, a way of specifying the overall N= inside the `group_count()` layer?

`set_format_strings(f_str("xx/xx (xxx.x)", distinct_n, N, distinct_pct))` ?

Thanks,
Robert

**Comment by elimillera** [comment]
Hello @Robert-Krajcik,

Thanks for the suggestion. This is a gap that has been identified previously and is on our radar. I'm expecting the next big update to Tplyr would include this. 

Right now you are able to get this from the `get_numeric_data()` function.
```
tab <- tplyr_table(safetyData::adam_adsl, TRT01P, where = SAFFL == "Y") %>% 
  add_layer(
    group_count(AGEGR1, by = "Age Categories n (%)")
  ) 

build(tab)

get_numeric_data(tab)[[1]]$total
```

**Comment by Robert-Krajcik** [comment]
Thanks for getting back to me, and showing me the workaround, @elimillera !

**Comment by mstackhouse** [comment]
This change is updated on the devel branch and merged from #39
